asrtoolkit/data_structures/corpus.py

```diff
@@ -57,6 +58,35 @@ def validate(self):
 
         return valid
 
+    def prepare_for_training(self, target, sample_rate=16000, nested=False):
```

Comment:
Not sure if the duplicate function name here is an issue or not, but it may be worthwhile keeping distinct names. I'm undecided.

Reply:
There's a duplicated function name in audio_file class as well

Reply:
This suggests an ABC